libs/core/src/components/sage-switch/test/sage-switch.e2e.ts

```diff
@@ -0,0 +1,83 @@
+import { newE2EPage } from '@stencil/core/testing';
+
+describe('sage-switch', () => {
+  it('renders a checked input when toggled', async () => {
+    const page = await newE2EPage();
+    await page.setContent('<sage-switch></sage-switch>');
+    const component = await page.find('sage-switch');
+    expect(component).toHaveClass('hydrated');
+
+    const el = await page.find('sage-switch >>> input');
+    let value = await el.getProperty('checked');
+    expect(value).toBe(false);
+
+    component.setProperty('checked', 'true');
+    await page.waitForChanges();
+    value = await el.getProperty('checked');
+    expect(value).toBe(true);
+  });
+
+  it('renders a disabled input when toggled', async () => {
+    const page = await newE2EPage();
+    await page.setContent('<sage-switch></sage-switch>');
+    const component = await page.find('sage-switch');
+    expect(component).toHaveClass('hydrated');
+
+    const el = await page.find('sage-switch >>> input');
+    let value = await el.getProperty('disabled');
+    expect(value).toBe(false);
+
+    component.setProperty('disabled', 'true');
+    await page.waitForChanges();
+    value = await el.getProperty('disabled');
+    expect(value).toBe(true);
+  });
+
+  it('renders an invalid input when toggled', async () => {
+    const page = await newE2EPage();
+    await page.setContent('<sage-switch></sage-switch>');
+    const component = await page.find('sage-switch');
+    expect(component).toHaveClass('hydrated');
+
+    const el = await page.find('sage-switch >>> input');
+    let value = el.getAttribute('aria-invalid');
+    expect(value).toBe(null);
+
+    component.setProperty('invalid', 'true');
+    await page.waitForChanges();
+    value = el.getAttribute('aria-invalid');
+    expect(value).toBe("true");
+  });
+
+  it('renders an invalid input with error message when toggled', async () => {
+    const page = await newE2EPage();
+    await page.setContent('<sage-switch invalid="true" error-message="Please correct this item"></sage-switch>');
+    const component = await page.find('sage-switch');
+    expect(component).toHaveClass('hydrated');
+
+    const el = await page.find('sage-switch >>> input');
+    const errText = await page.find('sage-switch >>> .sage-switch__message--error');
+    const ariaDesc = el.getAttribute('aria-describedby');
+    const ariaInvalid = el.getAttribute('aria-invalid');
+    expect(ariaInvalid).toBe("true");
```

Comment:
Prefer newline here to identify the separation of const assignments. One-to-one is fine, but multiple it can become easy to overlook.